phpstan.neon.dist

```diff
@@ -7,6 +7,8 @@ parameters:
     ignoreErrors:
         - '~Method [a-zA-Z0-9\\_]+::from[a-zA-Z0-9_]*\(\) has no return typehint specified.~'
         - '~Method [a-zA-Z0-9\\_]+Rate::[a-zA-Z0-9_]*\(\) has no return typehint specified.~'
+        - message: '~Strict comparison using === between int and false will always evaluate to false.~'
```

Comment:
Where is this actually happening?

Reply:
https://github.com/juliangut/rate-limit/blob/feature/apcu-adapter/src/ApcuRateLimiter.php#L84